reference/docs-conceptual/learn/remoting/wsman-unsupported-for-nonwindows.md

```diff
@@ -0,0 +1,26 @@
+---
+description: This article is explains the support policy for WSMan-based remoting on non-Windows platforms.
+ms.date: 06/22/2022
+title: WSMan remoting is not supported on non-Windows platforms
+---
+# WSMan remoting is not supported on non-Windows platforms
+
+Since the release of PowerShell 6, support for remoting over WS-Management (WSMan) on non-Windows
+platforms has only been available to a limited set of Linux distributions. All versions of those
+distributions that supported WSMan are no longer supported.
```

Comment:
Can we reword this to make it clear that the distro versions are not supported by their vendors, this isn't a change in support by us